mysql-test/mroonga/storage/create/table/column/flags/r/invalid_error.result

```diff
@@ -1,9 +1,9 @@
 CREATE TABLE tags (
 name VARCHAR(64) PRIMARY KEY COLLATE 'utf8mb4_general_ci'
-) DEFAULT CHARSET=utf8mb4;
+) DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
```

Comment:
Do we need this? It seems that `tags.name` has `COLLATE 'utf8mb4_general_ci'`. Can we use one of them?